src/System.Security.Cryptography.Csp/ref/System.Security.Cryptography.Csp.cs

```diff
@@ -7,6 +7,28 @@
 
 namespace System.Security.Cryptography
 {
+#if netcoreapp11
```

Comment:
Currently there is an issue with inconsistent contracts that I tried to work around with netcoreapp11 (exists in 462 contract but not in implementation causing genfacades to fail)... I'm working with @joperezr on that before I can commit this PR. It looks like just the AesCryptoServiceProvider type has the issue, so at a minimum that attribute will only exist on that type until we can get the contracts stuff fixed